Wikipedia/Code/UIImageView+WMFImageFetchingInternal.m

```diff
@@ -174,7 +174,7 @@ - (void)wmf_setImage:(UIImage *)image
 }
 
 - (void)wmf_setImage:(UIImage *)image
-       animatedImage:(nullable FLAnimatedImage *)animatedImage
+       animatedImage:(FLAnimatedImage *)animatedImage
```

Comment:
Should this and the one above still be marked nullable?

Reply:
@joewalsh it probably doesn't hurt, I was undoing the work I did in https://github.com/wikimedia/wikipedia-ios/pull/3449/files since fetching the lead image is closer to the old way now.